src/main/java/step2/StringCalculator.java

```diff
@@ -0,0 +1,88 @@
+package step2;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class StringCalculator {
+
+    private static final String DELIMITER = " ";
+    private Set<String> opSet = new HashSet<>();
+    private String text;
+    private String[] textArray;
+    private int result;
+
+    public void checkText(String text) {
+        if (text == null || text.isEmpty()) {
+            throw new IllegalArgumentException();
+        }
+    }
+
+    public void checkOperator(String[] textArray) {
+        for (int i = 1; i < textArray.length; i += 2) {
+            if (!opSet.contains(textArray[i])) {
+                throw new IllegalArgumentException();
+            }
+        }
+    }
+
+    public void setText(String text) {
+        checkText(text);
+        checkOperator(text.split(DELIMITER));
+        this.text = text;
+        this.textArray = text.split(DELIMITER);
+    }
+
+    public int getResult() {
+        return result;
+    }
+
+    public StringCalculator() {
+        opSet.add("+");
+        opSet.add("-");
+        opSet.add("*");
+        opSet.add("/");
+    }
+
+    public int add(int a, int b) {
+        return a + b;
+    }
+
+    public int subtract(int a, int b) {
+        return a - b;
+    }
+
+    public int multiply(int a, int b) {
+        return a * b;
+    }
+
+    public int divide(int a, int b) {
+        return a / b;
+    }
+
+    public int calculate() {
+        result = Integer.parseInt(textArray[0]);
+
+        for (int i = 1; i < textArray.length; i += 2) {
+            int temp = 0;
+            int nextNum = Integer.parseInt(textArray[i + 1]);
+
+            switch (textArray[i]) {
+                case "+":
+                    temp = add(result, nextNum);
+                    break;
+                case "-":
+                    temp = subtract(result, nextNum);
+                    break;
+                case "*":
+                    temp = multiply(result, nextNum);
+                    break;
+                case "/":
+                    temp = divide(result, nextNum);
+                    break;
+            }
+            result = temp;
+        }
+
+        return result;
+    }
```

Comment:
- result를 calculate 메서드 지역변수로 선언해도 되겠네요! 👍 
- temp 변수는 사용하지 않아도 되겠어요.
- switch문에 default문은 방어적 코딩을 위해 항상 추가해보도록 연습해보세요! 😀 
```suggestion
    public int calculate() {
        int result = Integer.parseInt(textArray[0]);

        for (int i = 1; i < textArray.length; i += 2) {
            int nextNum = Integer.parseInt(textArray[i + 1]);

            switch (textArray[i]) {
                case "+":
                    result = add(result, nextNum);
                    break;
                case "-":
                    result = subtract(result, nextNum);
                    break;
                case "*":
                    result = multiply(result, nextNum);
                    break;
                case "/":
                    result = divide(result, nextNum);
                    break;
                default:
                    throw new IllegalArgumentException();
            }
        }
```